tests/unit/test_airflow.py

```diff
@@ -165,22 +167,23 @@ def test_byo_training_config_all_args(sagemaker_session):
             'mini_batch_size': '256'},
         'Tags': [{'{{ key }}': '{{ value }}'}]
     }
-    assert training_config == expected_config
+    assert config == expected_config
 
 
 def test_framework_training_config_required_args(sagemaker_session):
-    tf = TensorFlow(entry_point="{{ entry_point }}",
-                    framework_version='1.10.0',
-                    training_steps=1000,
-                    evaluation_steps=100,
-                    role="{{ role }}",
-                    train_instance_count="{{ instance_count }}",
-                    train_instance_type="ml.c4.2xlarge",
-                    sagemaker_session=sagemaker_session)
+    tf = tensorflow.TensorFlow(
```

Comment:
One of the Python mantras is to be consistent https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

I am ok with you importing the estimator as it is right now or the way that we import all the estimators before.